nbconvert/templates/html/slides_reveal.tpl

```diff
@@ -211,13 +209,29 @@ require(
 
         Reveal.addEventListener('slidechanged', update);
 
-        var update_scroll = function(event){
-          $(".reveal").scrollTop(0);
-        };
-
-        Reveal.addEventListener('slidechanged', update_scroll);
+        function setScrollingSlide(scroll) {
+
+            if (scroll === "True") {
+              var h = $('.reveal').height() * 0.95;
+              var hpx = "" + h + "px";
+              $('section.present').find('section')
+                .filter(function() {
+                  return $(this).height() > h;
+                })
+                .css('height', hpx)
```

Comment:
Does it work to use a height like `95%`, or even [CSS calc](https://developer.mozilla.org/en-US/docs/Web/CSS/calc) (`calc(100% - 20px)`), rather than doing calculations in JS?